float64 radius 
sensor_msgs/NavSatFix location
---
sensor_msgs/NavSatFix[] relevant_waypoints